registry/config_unix.go

```diff
@@ -3,12 +3,22 @@
 package registry
 
 import (
+	"os"
+
 	"github.com/spf13/pflag"
 )
 
+func initCertsDir() string {
+	dir := os.Getenv("DOCKER_REGISTRY_CERTS_DIR")
```

Comment:
Thanks, applied your review. Let me know if the name of the flag works, or if you would prefer something less clunky.